ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/vr/RDXVRModeControls.java

```diff
@@ -91,19 +98,30 @@ public void render()
 
       if (ImGui.collapsingHeader("Visual Feedback"))
       {
-         ImGui.checkbox("Floating Video Panel", showFloatingVideoPanel);
-         if (showFloatingVideoPanel.get())
-         {
-            // Stereo checkbox is enabled and interactive
-            ImGui.checkbox("Stereo Enabled", useStereoVision);
-         }
-         else
+         if (ImGui.checkbox("Floating Video Panel", showFloatingVideoPanel))
          {
-            // Stereo checkbox is disabled (grayed out)
-            ImGui.beginDisabled();
-            ImGui.checkbox("Stereo Enabled", useStereoVision);
-            ImGui.endDisabled();
+            Collection<RDXPanel> baseUIPanels = RDXBaseUI.getInstance().getImGuiPanelManager().getPanels();
+            for (RDXPanel panel : baseUIPanels)
+            {
+               if (panel instanceof RDXRobotPerceptionVisualizersPanel perceptionVisualizersPanel)
+               {
+                  RDXROS2ImageMessageVisualizer leftColorVisualizer = perceptionVisualizersPanel.getZedLeftColorImageVisualizer();
```

Comment:
I was talking with @ds58 about removing RDXRobotPerceptionVisualizersPanel. I think it a bad abstraction because its robot specific. It'd be better to just check instance of `RDXROS2ImageMessageVisualizer` and check the name of it or something. Or setup this spaghetti logic at a higher level.

Idk... This is probably fine for now.